app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/ToolbarIcons.kt

```diff
@@ -52,14 +60,25 @@ fun ToolbarFirstVetIcon(onClick: () -> Unit, modifier: Modifier = Modifier) {
 
 @Composable
 fun ToolbarCrossSellsIcon(onClick: () -> Unit, modifier: Modifier = Modifier) {
+  var isRotated by remember { mutableStateOf(false) }
+  val fullRotation by animateFloatAsState(
+    targetValue = if (isRotated) 360f else 0f,
+    animationSpec = tween(1500, 50),
```

Comment:
Do we have specs from design about the easing that we want this rotation to have?
If not, I suppose the default `FastOutSlowInEasing` looks good enough?